lib/bindings/AMQPBinding.js

```diff
@@ -92,9 +92,7 @@ function start(callback) {
 
     let durable;
 
-    if (
-        amqpConfig.options && amqpConfig.options.durable
-    ) {
+    if (amqpConfig.options && amqpConfig.options.durable) {
```

Comment:
the code of this file was tainted, but no really changes?

Reply:
This due linting and/or prettier. I just ran the scripts defined in the package.json